src/locales/ar-SA.json

```diff
@@ -273,5 +273,9 @@
     "button_show": "Show me my collection",
     "success_claim": "Your POAP is on your address",
     "error_claim": "There was a problem minting the token"
+  },
+  "charts": {
+    "charts": "Charts",
+    "noVotesYet": "There are no votes to visualize yet."
```

Comment:
Hi @Zerquix18, thanks for your contribution! You don't need to add this to every translation file, only default.json is enough.

Reply:
Removed them for all files except english and spanish. :)

Reply:
Thanks. Though the Spanish translation should be added through translate.snapshot.org instead of here.

Reply:
Sure. I assume that has to be done if the pull request is merged, right?

Reply:
No, but you can do it of you like.